### ISSUE TITLE
Get all players' scoring, regardless if they are on a team or free agent

### ISSUE THREAD

**Issue by Seawhales** [kiernann/fflr#21]
The function team_roster() gets the scoring data for players on teams, but there is no function to get the projected and scoring data for players on the waiver wire. Could this be added?

**Comment by kiernann** [comment]
There's currently no way to show _just_ waiver wire players, but the `all_players()` function has `next_wk` column for projections. You can filter out players from `team_roster()` using the unique ID to get waiver wire players.

The `all_players()` default row limits were changed a little in 0.3.13, so check the documentation for that.

``` r
library(tidyverse)
library(fflr)
rosters <- bind_rows(team_roster(lid = 252353))
all_players(limit = NULL) %>% 
  filter(is.na(match(id, rosters$id))) %>% 
  select(week, first, last, pro, pos, fpts, next_wk, this_szn)
```

    #> # A tibble: 944 x 8
    #>     week first    last          pro   pos    fpts next_wk this_szn
    #>    <dbl> <chr>    <chr>         <fct> <fct> <dbl>   <dbl>    <dbl>
    #>  1    12 Justin   Tucker        Bal   PK     97      7.78     97  
    #>  2    12 Matt     Ryan          Atl   QB    178.    19.2     178. 
    #>  3    12 DJ       Chark Jr.     Jax   WR     77.4    8.66     77.4
    #>  4    12 Jamison  Crowder       NYJ   WR     60.5    6.96     60.5
    #>  5    12 Harrison Butker        KC    PK     82      8.33     82  
    #>  6    12 J.D.     McKissic      Wsh   RB     57.4    8.25     57.4
    #>  7    12 Darrell  Henderson Jr. LAR   RB     92      7.48     92  
    #>  8    12 Saints   D/ST          NO    DS     54.5    5.20     54.5
    #>  9    12 Devin    Singletary    Buf   RB     64.8    6.63     64.8
    #> 10    12 Nyheim   Hines         Ind   RB     85.5    7.68     85.5
    #> # … with 934 more rows

<sup>Created on 2020-11-25 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by Seawhales** [comment]
Is there a way to get the fantasy points scored by player per week?

**Comment by kiernann** [comment]
I don't believe so, but that's definitely a feature that should be added. Poking around the API, I can only get it to return six stats for a given player. For example, Dalvin Cook's stats have these totals (on Wednesday, pre week 12):

1. Last week = 20.00000
2. Week before that = 11.20000
3. This season (so far) = 216.30000
4. Last season = 225.15700 
5. Next week = 20.55676
6. Preseason projection = 239.40000

I'll have to poke around some more and see if there's an easy way to return all the weeks, like you would see on the website.

![Screenshot_2020-11-25 Fantasy Football Team Clubhouse - ESPN](https://user-images.githubusercontent.com/18021678/100264100-5ebd8300-2f1c-11eb-89d1-53b21f873d71.png)

**Comment by kiernann** [comment]
One workaround for rostered players would be to use `team_roster(week = 1)` and just change the week.

**Comment by Seawhales** [comment]
Thanks for your efforts. I'm really enjoying using this.
